app.ts

```diff
@@ -0,0 +1,462 @@
+const headings: any = document.querySelector("#Headings");
+const content_cols: any = document.querySelector("#Content");
+const pageStats: any = document.getElementById("pageStats");
+const nextButton: any = document.querySelector("#next");
+const prevButton: any = document.querySelector("#prev");
+const clear = "";
+let paramOne: any = "0";
```

Comment:
Use the `any` type sparingly. Check the javascript style guide for the preferences and standards that IMQS uses.